### PR TITLE
[CDAP-5892] - <my-ds-multiple-values/> widget in Hydrator++

### PR DESCRIPTION
**Pull request by ajainarayanan**
- Adds `<my-ds-multiple-values/>` widget to widget container.
- Modifies experimental feature to add a controller to give data to this directive.

### CODE REVIEW

File: cdap-ui/app/directives/widget-container/widget-ds-multiplevalues/widget-ds-multiplevalues.js [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+
+        // initializing
+        function initialize() {
+          var str = $scope.model;
+          $scope.properties = [];
+
+          if (!str) {
+            //intialize to one empty property
+            $scope.properties.push({
+              values: Array($scope.numValues).join('.').split('.')
```
Comment (elfenheart) [inline cdap-ui/app/directives/widget-container/widget-ds-multiplevalues/widget-ds-multiplevalues.js:53]: what is this intending to do?

Comment (ajainarayanan) [inline cdap-ui/app/directives/widget-container/widget-ds-multiplevalues/widget-ds-multiplevalues.js:53]: Create an array of size  `$scope.numValues` with empty strings.

`Array(4).join('.').split('.') == ['', '', '', '']`

**Comment by elfenheart** [comment]
The focus when you add a new line is on the last value field. Can we make it to be the first?

**Comment by ajainarayanan** [comment]
@elfenheart Added the focus to the first input.

**Comment by elfenheart** [comment]
👍  LGTM.

Side note: Might have to add this new widget to the Hydrator widget Wiki

**Comment by ajainarayanan** [comment]
@elfenheart Added a [JIRA](https://issues.cask.co/browse/CDAP-5976) to track this. Will create a separate PR for it. Merging this as of now.

**Pull request merged**